src/DIRAC/WorkloadManagementSystem/Client/PoolXMLSlice.py

```diff
@@ -5,21 +5,17 @@
 """
```

Comment:
Why are there changes in this file?

Reply:
Just some other simplifications (they are all in a different commit)